.cicd/docker-hash.sh

```diff
@@ -0,0 +1,20 @@
+export IMAGE_TAG=${IMAGE_TAG:-'ubuntu-18.04'}
+
+function determine-hash()
+{
```

Comment:
Why have { on a separate line?

Reply:
Because when the `{` and `}` characters are aligned in the same column, it makes it more clear where scope begins, particularly in larger files or with longer function signatures.